### ISSUE TITLE
[BUG] DK: Quest The Light of Dawn 

### ISSUE THREAD

**Issue by WoW-Mercury** [mangosR2/scriptdev2#146]
The mission of this event is not working properly.
The Light of Dawn: http://www.wowhead.com/quest=12801/the-light-of-dawn

Video on our server: http://www.youtube.com/watch?v=4D-M1LkXFXA
Video on official: http://www.youtube.com/watch?v=C3rBbv7OOAI

Thanks

**Comment by ukulutl** [comment]
as I saw, event works but when tirion appears darion run away and event becomes buged, if darion stay at the correct place maybe works fine

**Comment by michalpolko** [comment]
You have corrupted waypoints in sd2 table for Darion.

**Comment by raynar** [comment]
mmm @Michalpolko , could you post here your waypoints for Darion please?

**Comment by michalpolko** [comment]
Waypoints are in repository.
I had similiar problem on my server some time ago, Darion was moving to some weird points and then event was freezing at some point. It appeared that some waypoints are missing in the DB (maybe overwritten by some other script).

**Comment by raynar** [comment]
this file? 
https://github.com/mangosR2/scriptdev2/blob/master/sql_mr/mr00161_scriptdev2_ebon_hold_dk_quest.sql

**Comment by WoW-Mercury** [comment]
I think I found the patch to fix the bug in the mission of the Light of Dawn
Link: http://pastebin.com/6ADwLrRk

**Comment by Undergarun** [comment]
@WoW-Mercury ¿De dónde los has sacado? Para poder poner al autor si lo commiteo.

**Comment by WoW-Mercury** [comment]
@Undergarun De aqui: https://github.com/przemratajczak/scriptdev2/blob/master/DB_Updates/ScriptDev2/s12_sd2_Q_LightOfDawn.sql
Al final del todo

**Comment by WoW-Mercury** [comment]
@Undergarun Perdon de aqui: https://github.com/przemratajczak/scriptdev2/blob/master/DB_Updates/ScriptDev2/s12_sd2_Q_LightOfDawn.sql

**Comment by WoW-Mercury** [comment]
My patch don't work. Any volunteers to try and make more corrections

**Comment by Circalel** [comment]
After patch sd2_mr0491 NPC: Mograine stops before reaching the site where you have to stop

http://tracker.wow-mercury.com/index.php?getfile=102

Thanks

**Comment by Venom** [comment]
yes. same problem.  mograne runing awway like a chicken when tirion  comes out. some times not even runs to battle place. and just standing there.

**Comment by rsa** [comment]
test after 2761, please
